### DISCUSSION TITLE
Ultraschallsensor

### DISCUSSION THREAD

**Discussion by sirtiger** [diefenbecker/opencistern#4] [Ideas]
Hi,

ein sehr schönes Projekt. Sieht echt gut aus.
Ich habe es damals mit einer Siemens Logo + Display für das Esszimmer und einem Arduino + 7 Segment Anzeige in der Werkstatt realisiert.

Aus der Vergangenheit kann ich Dir nur den Tipp geben, einen anderen US Sensor zu benutzen. Da es in der Zisterne immer schön feucht ist, ist mir bei zwei US Sensoren (HC-SR04)  schon jeweils der silberne Spiegel der Membran abgegammelt.
Ich benutze seitdem folgenden US Sensor. -> Wasserdichte Ultraschall Modul JSN-SR04T / AJ-SR04M
[https://de.aliexpress.com/item/4001167945241.html?pdp_npi=2%40dis%21EUR%21€%204%2C42%212%2C20%20€%21%21%21%21%21%40211b801716759676134323387ea16e%2110000015332765689%21btf&_t=pvid%3Ae1d2e255-38a2-4979-8b2b-6ac3712ec7eb&afTraceInfo=4001167945241__pc__pcBridgePPC__xxxxxx__1675967613&spm=a2g0o.ppclist.product.mainProduct&gatewayAdapt=glo2deu](url)

Bis dann
Timo

**Comment by diefenbecker** [comment]
> Hi,
> 
> ein sehr schönes Projekt. Sieht echt gut aus.
> Ich habe es damals mit einer Siemens Logo + Display für das Esszimmer und einem Arduino + 7 Segment Anzeige in der Werkstatt realisiert.
> 
> Aus der Vergangenheit kann ich Dir nur den Tipp geben, einen anderen US Sensor zu benutzen. Da es in der Zisterne immer schön feucht ist, ist mir bei zwei US Sensoren (HC-SR04)  schon jeweils der silberne Spiegel der Membran abgegammelt.
> Ich benutze seitdem folgenden US Sensor. -> Wasserdichte Ultraschall Modul JSN-SR04T / AJ-SR04M
> [https://de.aliexpress.com/item/4001167945241.html?pdp_npi=2%40dis%21EUR%21€%204%2C42%212%2C20%20€%21%21%21%21%21%40211b801716759676134323387ea16e%2110000015332765689%21btf&_t=pvid%3Ae1d2e255-38a2-4979-8b2b-6ac3712ec7eb&afTraceInfo=4001167945241__pc__pcBridgePPC__xxxxxx__1675967613&spm=a2g0o.ppclist.product.mainProduct&gatewayAdapt=glo2deu](url)
> 
> Bis dann
> Timo
> 

Hi Timo,

im Blog wurde das Thema schon oft diskutiert. Der JSN-SR04T ist ja kompatibel mit dem HC-SR04 und kann ohne Anpassungen genutzt werden  Ich aktualisiere die Sensoren im GIT noch entsprechend.. Danke aber für den Hinweis.

Gruß
Chris
